mcp_server/src/services/dynamic_registry/automatic_registration_manager.ts

```
@@ -0,0 +1,412 @@
+        break;
+
+      case "activeLocationChanged":
+        this.logger.info(
+          "[AutoRegistration] Active location changed, attempting registration"
+        );
+        await this.attemptRegistration("active_location_changed");
+        break;
+
+      case "debugSessionStopped":
+        // Clean up when debug session stops
+        const sessionId = event.data?.debugSessionId;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)